tpm2/test/tpm2_test.go

```diff
@@ -2111,3 +2111,72 @@ func TestPolicyOr(t *testing.T) {
 		t.Errorf("PolicyOr with error: %v", err)
 	}
 }
+
+func TestNVUndefineSpaceSpecial(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+
+	// Set up policy session for creation in a trial session
+	sess, _, err := StartAuthSession(rw, HandleNull, HandleNull, make([]byte, 16), nil, SessionTrial, AlgNull, AlgSHA256)
+	if err != nil {
+		t.Errorf("StartAuthSession() for policy generation failed: %v", err)
+	}
+
+	// Use PolicyCommandCode(rw, sess, CmdNVUndefineSpaceSpecial) to manipulate the session
```

Comment:
I think the policy is now so simple that this comment is not needed 😄

I would actually propose replacing the comments on lines 2119, 2125, 2130, and 2135 with one comment before the StartAuthSession that just says "use a trial session to calculate the PolicyCommandCode policy" (or similar), then removing blank lines so that that logical step is grouped nicely into a paragraph of one logical step.

Reply:
done